.eslintrc.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
     './packages/eslint-config-sanity/import.js',
     'plugin:@typescript-eslint/recommended',
     'plugin:react-hooks/recommended',
+    'plugin:cypress/recommended',
```

Comment:
This should be moved to a file in `cypress/.eslintrc`, so that the cypress-specific eslint rules don't affect all source files in the monorepo:

```
{
  "plugins": ["cypress"],
  "extends": ["plugin:cypress/recommended"]
}
```

Reply:
Very good point! :grinning: Moved in f1e7fe7f3